handbook/company/pricing-features-table.yml

```diff
@@ -613,15 +613,16 @@
   tier: Premium
   jamfProHasFeature: yes
   jamfProtectHasFeature: yes
-- industryName: Interactive MDM migration #   « end-user initiated MDM migration, with interactive UI
+- industryName: MDM migration #   « end-user initiated MDM migration, with interactive UI
```

Comment:
We have automated migration if you control your current MDM solution data, and we have interactive if you do not. I suggest renaming the feature to “MDM migration” to capture this in a single feature line item.